core/src/block_creation_loop.rs

```diff
@@ -29,6 +30,7 @@ use {
         bank::{Bank, NewBankOptions},
         bank_forks::BankForks,
     },
+    stats::{BlockCreationLoopMetrics, SlotMetrics},
```

Comment:
I vote for consistency on the type names. Maybe shorten BlockCreationLoopMetrics to just `LoopMetrics`. `BlockCreation` part is sort of implied by where it lives